internal/querycoord/querynode.go

```diff
@@ -41,7 +41,7 @@ type queryNode struct {
 }
 
 func newQueryNode(address string, id UniqueID, kv *etcdkv.EtcdKV) (*queryNode, error) {
-	client, err := nodeclient.NewClient(address, 3*time.Second)
+	client, err := nodeclient.NewClient(context.TODO(), address, 300)
```

Comment:
1. context should be input arguments of `newQueryNode`, not use `context.TODO()`
2. `300` is magic number, what's it means?